src/main/kotlin/no/skatteetaten/aurora/boober/feature/azure/AzureFeature.kt

```diff
@@ -25,7 +25,9 @@ import no.skatteetaten.aurora.boober.service.CantusService
 @Service
 class AzureFeature(
     cantusService: CantusService,
-    @Value("\${clinger.sidecar.default.version:0.4.0}") val sidecarVersion: String
+    @Value("\${clinger.sidecar.default.version:0.4.0}") val sidecarVersion: String,
```

Comment:
Jeg ville droppet default her. Vet du ikke har endret det akkurat nå, men det gir vel ikke meningen med en slik versjon som default? Om vi glemmer å confe det, så får vi gammel versjon?

Reply:
Nå du nevner det, ja, dette er korrekt. Kan være aktuelt å sette en "1" versjon, kanskje, i stedet for å spesifisere helt til patchlevel.

Reply:
Eller alltid kreve config, men man kan jo kanskje skjele til hva som er vanlig for Boober... Hva som er gjort med fluentbit, for eksempel.

Reply:
Jeg endte med å legge inn 0 som default-versjon. Dette er pt. den nyeste major-versjon av clinger.